java/arcs/core/entity/BaseHandle.kt

```diff
@@ -20,8 +20,12 @@ abstract class BaseHandle<T : Entity>(
 ) : Handle {
     protected var closed = false
 
-    override suspend fun onSync(action: () -> Unit) = storageProxy.addOnSync(name, action)
+    override suspend fun onReady(action: () -> Unit) = storageProxy.addOnReady(name, action)
 
+    // TODO: temporary alias
+    override suspend fun onSync(action: () -> Unit) = onReady(action)
```

Comment:
Mark with `@Deprecated` annotation?

Reply:
I didn't bother because as soon as this lands I'm going to update the google side, then remove it from here.